mne/io/eeglab/eeglab.py

```diff
@@ -220,9 +235,20 @@ def _set_dig_montage_in_init(self, montage):
         )
 
 
+def _handle_montage_units(montage_units):
+    n_char_unit = len(montage_units)
+    if not (0 < n_char_unit < 3) or not montage_units[-1] == 'm':
```

Comment:
If you want something shorter
```suggestion
    if montage_units[-1:] != 'm' or n_char_unit > 2:
```